src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletControllerTest.java

```diff
@@ -186,6 +186,29 @@ void itShouldReturnBadRequestWhenIdIsZero() throws Exception {
         editResultActions.andExpect(status().isBadRequest());
     }
 
+    @Test
+    void itShouldReturnBadRequestWhenIdIsNegative() throws Exception {
+        //GIVEN
+        TestEditWalletDto editWalletDto = returnTestEditWalletDtoWithIdNegative();
+        given(walletService.updateWallet(Mockito.any())).willReturn(
+                new WalletDTO(
+                        Mockito.anyLong(),
+                        Mockito.anyString(),
+                        Instant.now()
+                )
+        );
+
+        //WHEN
+        ResultActions editResultActions = mockMvc.perform(
+                patch("/api/wallet")
```

Comment:
Why `patch()` method is used for testing, while in `WalletController.java` only Put REST API method is defined?

Did you run those tests cases? Becasue i'm pretty sure that they failed.